src/main/resources/db.properties

```diff
@@ -5,4 +5,4 @@ db.username = root
 db.password = Vi4ft5q7Ktrr9T6toPcN
 # Hibernate properties
 hibernate.show_sql = true
-hibernate.hbm2ddl.auto = create-drop
+hibernate.hbm2ddl.auto = update
```

Comment:
why have you decided to change this?

Reply:
testing with empty base is tiring :) Done